READMEs/get-ocean-on-sapphire.md

```diff
@@ -0,0 +1,58 @@
+# Get OCEAN on Sapphire Mainnet
+
+## 0. Steps
+
+- [1. Get OCEAN on Eth Mainnet](#1-get-ocean-on-eth-mainnet)
+        - [1. Create a Wallet](#1-create-a-wallet)
+        - [2. Purchase OCEAN](#2-purchase-ocean)
+- [2. Transfer OCEAN to Sapphire Mainnet via Celer](#2-transfer-ocean-to-sapphire-mainnet-via-celer)
+        - [1. Navigate to Celer bridge](#1-navigate-to-celer-bridge)
+        - [2. Connect Your Wallet](#2-connect-your-wallet)
+        - [3. Configure Transfer Settings](#3-configure-transfer-settings)
+        - [4. Execute the Transfer](#4-execute-the-transfer)
+- [3. Verify Your Tokens on Sapphire Mainnet](#3-verify-your-tokens-on-sapphire-mainnet)
+- [Conclusion](#conclusion)
+
+## 1. Get OCEAN on Eth Mainnet
+
+### 1. Create a Wallet
```

Comment:
Change `1.` to `1.1`.

Similar for others, for 1.2, 2.1, 2.2, etc.

Why: much easier to understand context.